cloudinit/config/modules.py

```diff
@@ -198,6 +202,14 @@ def _fixup_modules(self, raw_mods) -> List[ModuleDetails]:
                 # Misconfigured in /etc/cloud/cloud.cfg. Reset so cc_* module
                 # default meta attribute "frequency" value is used.
                 freq = None
+            if mod_name in RENAMED_MODULES:
+                LOG.warning(
```

Comment:
think this should be a `LOG.deprecated()`?

Reply:
Good thought. Done